src/main/java/com/uber/jenkins/phabricator/InlineBuilder.java

```diff
@@ -0,0 +1,77 @@
+package com.uber.jenkins.phabricator;
```

Comment:
All new files (and existing files) need license headers -- if you're using IntelliJ I can show you my settings. Important since this is MIT licensed and open source.